src/media/js/addon/containers/dashboardDetail.js

```diff
@@ -71,7 +71,12 @@ export class AddonDashboardDetail extends React.Component {
 
         {!this.props.addon.deleted &&
           <div>
-            <AddonForDashboardDetail {...this.props.addon}/>
+            <Provider store={this.context.store}>
```

Comment:
i don't think you need to wrap it in a provider. i did it for the versionlistingcontainer because the hierarchy got too deep and i wanted to split it to another smart component.